recipes/nebari-dask/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "2022.10.1" %}
+
+package:
+  name: nebari-dask
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - dask
+    - distributed ==2022.04.2
+    - dask-gateway ==2022.4.0
+    - click ==8.0.2
+
+test:
+  imports:
+    - dask
+
+about:
+  home: https://github.com/nebari-dev
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: "Nebari dask dependencies."
+
+extra:
+  recipe-maintainers:
+    - viniciusdc
+    - iameskild
+    - nebari-sensei
```

Comment:
According to the description, this is a bot account. Not sure what our policy on that is.

Reply:
I can remove that... but I would like to know if its possible to keep a team in the feedstock repo in the maintainer list.. would that work?

Reply:
If we need more folks - you can add me @viniciusdc